web-dev/src/components/LandingPage/index.js

```diff
@@ -0,0 +1,204 @@
+import React from "react";
+import "./landing-page.css"
+import { Link } from "react-router-dom";
+import LandingBackground from "./landing-background.jpg"
```

Comment:
Can you move all of the images into the public/images folder? You can make your own subdirectory if you want.